community/contribute/README.md

```diff
@@ -15,3 +15,15 @@ To submit an RFP, head to the [RFP forum category](https://forum.pokt.network/c/
 A PEP is a type of proposal that requests a POKT grant from the DAO treasury. You could get a POKT grant for contributions to the Pocket Network ecosystem.
 
 Read more about [submitting a proposal](../governance/proposals.md).
+
+### Submit a PR - Become an Active Pocket V1 Contributor
+
+You can become an open-source contributor by submitting code contributions to the [core Pocket repository](https://github.com/pokt-network/pocket) or the [specifications](https://github.com/pokt-network/pocket-network-protocol).
```

Comment:
I think the phrase "core Pocket repository" is confusing, considering that there is a separate `pocket-core` repo as well. Is there another way we could refer to this?

In addition, I know you mentioned it in the header, but I think mentioning "v1" somewhere in here would make it clearer as to what you're going for here.

Reply:
Done. See the updated change and resolve if it looks good to you.

Reply:
I think that's fine.